docker/README.adoc

```diff
@@ -198,6 +198,9 @@ The following environment variables will populate the default `killbill.properti
 * `KILLBILL_CACHE_CONFIG_REDIS_URL` (default `redis://127.0.0.1:6379`)
 * `KILLBILL_CACHE_CONFIG_REDIS_PASSWORD` (default `null`)
 * `KILLBILL_CACHE_CONFIG_REDIS_CONNECTION_MINIMUM_IDLE_SIZE` (default `1`)
+* `KILLBILL_ANALYTICS_REFRESH_DELAY` (default `` seconds)
```

Comment:
Default is actually hardcoded at 10.

Reply:
fixed in https://github.com/killbill/killbill-cloud/pull/190/commits/ac4870abc9c352e6b1be258c182dbf305cff131d